langkit/lexer/__init__.py

```diff
@@ -879,7 +879,7 @@ def __init__(self, match_length, *alts):
             self.location = extract_library_location()
             self.match_length = match_length
 
-            for i, alt in enumerate(alts):
+            for _, alt in enumerate(alts):
```

Comment:
Let’s remove the call to enumerate altogether

Reply:
Right ... :derp: